anesthetic/samples.py

```diff
@@ -977,7 +978,7 @@ def recompute(self, logL_birth=None, inplace=False):
                               "\nDropping the invalid samples." %
                               (n_bad, len(samples), n_equal),
                               RuntimeWarning)
-                samples = samples[~invalid].reset_index(drop=True)
+                samples = samples.loc[~invalid].reset_index(drop=True)
```

Comment:
`.loc` can go...

Reply:
I think I've put back/`to_numpy()`ed all the ones I changed.